stl/inc/memory

```diff
@@ -3242,7 +3242,7 @@ public:
         return _Mypair._Get_first();
     }
 
-    _NODISCARD add_lvalue_reference_t<_Ty> operator*() const noexcept /* strengthened */ {
+    _NODISCARD add_lvalue_reference_t<_Ty> operator*() const noexcept(noexcept(*declval<pointer>())) {

```

Comment:
We have to fully-qualify all calls to non-`_Ugly` functions to avoid users hijacking them via ADL, yes, even `declval`:

```suggestion
    _NODISCARD add_lvalue_reference_t<_Ty> operator*() const noexcept(noexcept(*_STD declval<pointer>())) {
```